Index: Config/DefaultGame.ini
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>[/Script/EngineSettings.GeneralProjectSettings]\r\nProjectID=12BA3E324B13564816B40EB0FA78BF09\r\nProjectName=OdanHodo\r\nProjectDisplayedTitle=NSLOCTEXT(\"[/Script/EngineSettings]\", \"F939C98D4FEB6EB8A22918A000D946B4\", \"横断歩道操作すれば怖くない\")\r\nProjectDebugTitleInfo=NSLOCTEXT(\"[/Script/EngineSettings]\", \"D20744044E80A52F220597BB6A541DBF\", \"横断歩道操作すれば怖くない -Debug板-\")\r\nDescription=横断歩道操作すれば怖くないのゲームプロジェクト\r\nCopyrightNotice=Copyright (c) 2024, Noirkun All rights reserved.\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Config/DefaultGame.ini b/Config/DefaultGame.ini
--- a/Config/DefaultGame.ini	
+++ b/Config/DefaultGame.ini	
@@ -6,3 +6,15 @@
 Description=横断歩道操作すれば怖くないのゲームプロジェクト
 CopyrightNotice=Copyright (c) 2024, Noirkun All rights reserved.
 
+[/Script/Engine.AssetManagerSettings]
+-PrimaryAssetTypesToScan=(PrimaryAssetType="Map",AssetBaseClass=/Script/Engine.World,bHasBlueprintClasses=False,bIsEditorOnly=True,Directories=((Path="/Game/Maps")),SpecificAssets=,Rules=(Priority=-1,ChunkId=-1,bApplyRecursively=True,CookRule=Unknown))
+-PrimaryAssetTypesToScan=(PrimaryAssetType="PrimaryAssetLabel",AssetBaseClass=/Script/Engine.PrimaryAssetLabel,bHasBlueprintClasses=False,bIsEditorOnly=True,Directories=((Path="/Game")),SpecificAssets=,Rules=(Priority=-1,ChunkId=-1,bApplyRecursively=True,CookRule=Unknown))
++PrimaryAssetTypesToScan=(PrimaryAssetType="Map",AssetBaseClass="/Script/Engine.World",bHasBlueprintClasses=False,bIsEditorOnly=False,Directories=((Path="/Game/Maps")),SpecificAssets=("/Game/Game/Map/TitleMap.TitleMap","/Game/Game/Map/GameMap.GameMap","/Game/Game/Map/GameMap1.GameMap1","/Game/Game/Map/SelectStageMap.SelectStageMap"),Rules=(Priority=-1,ChunkId=-1,bApplyRecursively=True,CookRule=Unknown))
++PrimaryAssetTypesToScan=(PrimaryAssetType="PrimaryAssetLabel",AssetBaseClass="/Script/Engine.PrimaryAssetLabel",bHasBlueprintClasses=False,bIsEditorOnly=True,Directories=((Path="/Game")),SpecificAssets=,Rules=(Priority=-1,ChunkId=-1,bApplyRecursively=True,CookRule=Unknown))
+bOnlyCookProductionAssets=False
+bShouldManagerDetermineTypeAndName=False
+bShouldGuessTypeAndNameInEditor=True
+bShouldAcquireMissingChunksOnLoad=False
+bShouldWarnAboutInvalidAssets=True
+MetaDataTagsForAssetRegistry=()
+
